src/Console/Output/ErrorOutput.php

```diff
@@ -86,7 +87,7 @@ public function listErrors(string $process, array $errors): void
                 $this->output->writeln('');
                 $stackTrace = $e->getTrace();
                 foreach ($stackTrace as $trace) {
-                    if (isset($trace['class']) && \Symfony\Component\Console\Command\Command::class === $trace['class'] && 'run' === $trace['function']) {
+                    if (isset($trace['class']) && Command::class === $trace['class'] && 'run' === $trace['function']) {
```

Comment:
can you describe why we needed to add isset?
it was some warning in phpstan, or we changed the logic somewhere?

Reply:
`isset()` was already there, the only changes in this PR are: ruleset reconfiguration (manual) and applying CS fixes (automated). Here FQCN was shortened, nothing more.

Reply:
🤦🏻 with split-view, I misseen the check as it was visually broken to line above 🤦🏻 

![Screenshot 2023-12-30 at 01 18 08](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/3c6fd469-f7d5-4e2b-9b94-eff45e4120d6)